server/routes/sessions.js

```diff
@@ -44,20 +44,86 @@ const sessions = [
         sport: "Soccer",
         joined: false,
         dateTime:" dayjs('2022-04-17T15:30')",
-        type: "outdoor,"
+        type: "outdoor"
     },
+    {
+      name: "Basketball",
+      description: "play time",
+      city: "Vancouver",
+      location: "MacInnes Field",
+      equipment: "Frisbee",
+      playersNeeded: 3,
+      groupId: 3,
+      image: image,
+      sport: "Basketball",
+      joined: false,
+      dateTime: "dayjs('2022-04-17T15:30')"
+  },
+  {
+      name: "Badminton",
+      description: "play time",
+      city: "Vancouver",
+      location: "MacInnes Field",
+      equipment: "Frisbee",
+      playersNeeded: 3,
+      groupId: 4,
+      image: image,
+      sport: "Badminton",
+      joined: true,
+      dateTime: "dayjs('2022-04-17T15:30')"
+  },
+  {
+      name: "Tennis",
+      description: "play time",
+      city: "Vancouver",
+      location: "MacInnes Field",
+      equipment: "Frisbee",
+      playersNeeded: 3,
+      groupId: 5,
+      image: image,
+      sport: "Tennis",
+      joined: true,
+      dateTime: "dayjs('2022-04-17T15:30')"
+  },
+  {
+      name: "Football",
+      description: "play time",
+      city: "Vancouver",
+      location: "MacInnes Field",
+      equipment: "Frisbee",
+      playersNeeded: 3,
+      groupId: 6,
+      image: image,
+      sport: "Soccer",
+      joined: false,
+      dateTime: "dayjs('2022-04-17T15:30')"
+  },
+  {
+      name: "Water Polo",
+      description: "play time",
+      city: "Vancouver",
+      location: "MacInnes Field",
+      equipment: "Frisbee",
+      playersNeeded: 3,
+      groupId: 7,
+      image: image,
+      sport: "Water Polo",
+      joined: true,
+      dateTime: "dayjs('2022-04-17T15:30')"
+  }
     
 ]
 
 router.get('/', function (req, res, next) {
+  console.log("i am here")
   return res.status(200).send(sessions);
 });
 
 router.get('/filter', function (req, res, next) {
 	const filterName = req.query.filter;
 	if (filterName === '')
-		return res.send(defaultItems)
-	let filterArray = defaultItems.filter(item => item.type === filterName);
+		return res.send(sessions)
+	let filterArray = sessions.filter(item => item.type === filterName);
 	return res.status(200).send(filterArray);
   });
```

Comment:
`filter` shouldn't be its own endpoint btw. It breaks REST principles. Filters should go into the query fields for endpoint calls